### PR TITLE
Move src/ to whatwg.org/

### PR DESCRIPTION
**Pull request by foolip**
Also start deploying to new server to prepare for the switch.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -14,6 +14,6 @@ else
     ssh-add deploy_key
     echo "$SERVER $SERVER_PUBLIC_KEY" > known_hosts
     echo "$NEW_SERVER $NEW_SERVER_PUBLIC_KEY" >> known_hosts
-    rsync --archive --verbose --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" src/ $DEPLOY_USER@$SERVER:whatwg.org/
-    rsync --archive --chmod=D755,F644 --verbose --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" idea.whatwg.org images.whatwg.org n.whatwg.org deploy@$NEW_SERVER:/var/www/
+    rsync --archive --verbose --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" whatwg.org/ $DEPLOY_USER@$SERVER:whatwg.org/
```
Comment (foolip) [inline deploy.sh]: Deliberately not simplifying this, trailing slashes in rsync make for fun mistakes.

**Comment by annevk** [comment]
Should have fixes #42 in commit message. LGTM. Might be a little annoying to rebase other PR on top of this?

**Comment by foolip** [comment]
I can merge the other PR into this one, should resolve everything with the magic of git.

**Comment by annevk** [comment]
Well we can't land the other PR yet, but I guess it'll be fine.

**Comment by foolip** [comment]
Yep, I'll handle any conflicts.

**Pull request merged**